appveyor.yml

```diff
@@ -0,0 +1,44 @@
+max_jobs: 1
+
+image: Visual Studio 2019
+
+cache:
+  - '%LocalAppData%\NuGet\v3-cache'
+
+nuget:
+  account_feed: false
+  project_feed: false
+  disable_publish_on_pr: true
+
+install:
+  - cmd: dotnet tool restore
```

Comment:
now we use the `.config/dotnet-tools.json` to specify which tools need to be installed locally.